src/askem_beaker/contexts/mira_model_edit/agent.py

```diff
@@ -87,84 +89,12 @@ async def add_template(self, name: str, subject: str, outcome: str, expr: str, m
             expr (str): the mathematical rate law for the transition.
             name (str): the name of the transition
         """
-        code = agent.context.get_code("add_template", {"model": model, "subject": subject, "outcome": outcome, "expr": expr, name: "name"})
-        return json.dumps(
+        code = agent.context.get_code("add_template", {"model": model, "subject": subject, "outcome": outcome, "expr": expr, "name": name})
+        loop.set_state(loop.STOP_SUCCESS)
+        return json.dumps( # <--- Is this returning to archytas? Why does this action not mean it sends a response with code_cell
```

Comment:
Are we doing something about the comment here?

Reply:
Thanks for catching